usaspending_api/etl/management/load_base.py

```diff
@@ -0,0 +1,345 @@
+from datetime import datetime
```

Comment:
This file contain common code to be potentially reused by other loaders correct?